examples/fnv.rs

```diff
@@ -0,0 +1,19 @@
+extern crate fnv;
+
+fn main() {
+
+use fnv::FnvHashMap;
+
+let mut map = FnvHashMap::default();
+map.insert(1, "one");
+map.insert(2, "two");
+map.insert(3, "three");
+
+for (number, word) in map.iter() {
+        println!("Number {}: {}", number, word);
+}
+
+map.remove(&(2));
+println!("The length of HashMap is {}.", map.len());
+println!("The first element is {}.", map.get(&(1)).unwrap());
+}
```

Comment:
The inside of this function needs to be intended one level, and likewise in the markdown.